statesman-publisher/src/main/java/io/appform/statesman/publisher/http/HttpUtil.java

```diff
@@ -25,33 +27,69 @@ public static OkHttpClient defaultClient(final String clientName,
                                              final MetricRegistry registry,
                                              final HttpClientConfiguration configuration) {
 
-        int connections = configuration.getConnections();
-        connections = connections == 0 ? 10 : connections;
 
-        int idleTimeOutSeconds = configuration.getIdleTimeOutSeconds();
-        idleTimeOutSeconds = idleTimeOutSeconds == 0 ? 30 : idleTimeOutSeconds;
+        try {
+            // Create a trust manager that does not validate certificate chains
```

Comment:
This is just to make the API calls from local. Will not be merging, but just posting here for reference